### ISSUE TITLE
why are some property names in declarations quoted and others not quoted?

### ISSUE THREAD

**Issue by macMikey** [macMikey/LCB-missing-manual#27]
i don't see a pattern to them.
is there any harm to quoting all of them?

**Comment by bwmilby** [comment]
Can you point to an example?  I don't think a quoted string would be interchangeable in LCB - there isn't a string literal concept like in LCS with strict compilation turned off.

**Comment by macMikey** [comment]
There are a bunch, and they're easy to find if you are using Atom or ST:
In the bundle, navigate down to the Extensions->widgets folder, and do a "find in folder" for 
`property "`
navbar, for instance has `"opaque"` and `"tooltip"`
header has "theme", "label", "showLabel", and "opaque"
switchButton has "theme" and "hilight"
svgpath has "scaledWidth", "scaledHeight", "flipped", and "angle"

**Comment by bwmilby** [comment]
From testing I can infer that any property where the name is also a keyword (i.e. highlight, angle) must be quoted as a property name.  If the token is not reserved, then quoting the property identifier is optional.

**Comment by macMikey** [comment]
did you see any expression for the `Name: Identifier` anywhere? I have not seen one.
I presumed that's why properties were quoted, but I couldn't find any discussion, so far, anyway.

**Comment by bwmilby** [comment]
Yes, at the top of the LCB Language Reference are regex expressions for all tokens.  I have not seen any discussion on it either, but tested svgpath by removing the quotes and got errors on the unquoted keywords.

**Comment by macMikey** [comment]
right, but it doesn't include quotes, right?
`Identifier: [A-Za-z_][A-Za-z0-9_.]*`

**Comment by bwmilby** [comment]
Correct, an identifier is not quoted.  I can only infer that this is an undocumented workaround to allow properties to exist that are also keywords.  The `PropertyDefinition` syntax needs to be updated to include the optional quote and the discussion needs to explain when it may be required.

**Comment by macMikey** [comment]
since we're annotating the manual, what do you think would be the best way to do that? If the manual gets updated, any annotations we make will be lost, right?

**Comment by macMikey** [comment]
in the [property page](https://github.com/macMikey/LCB-missing-manual/wiki/properties) I tried to address this, but I didn't modify the manual.
you have collaborator privs in the wiki, if you want to take a crack at it.

**Comment by bwmilby** [comment]
In the widget course, they seem to quote the property name in just about every case.
